test/integration/targets/callback_default/inventory

```diff
@@ -5,6 +5,6 @@ testhost ansible_connection=local ansible_python_interpreter="{{ ansible_playboo
 testhost5 ansible_host=169.254.199.200  # no connection is ever established with this host
 
 [nonlockstep]
-testhost10 i=1.0 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
-testhost11 i=2.0 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
-testhost12 i=3.0 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
+testhost10 i=0.5 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
+testhost11 i=3.0 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
+testhost12 i=12.0 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
```

Comment:
Could we dial these down a bit just to reduce execution time? 

```suggestion
# Sleep times were calculated as follows assuming 0.5s task overhead. First host does not need a sleep.
# (previous_task_sleep + 0.5) * number_of_tasks
testhost10 i=0 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
testhost11 i=1.5 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
testhost12 i=6 ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
```

Reply:
I can play with it, but I also want to get this merged ASAP to avoid continued CI issues over the weekend.  I can leave it on my to do list to shorten it, but at length of roughly 65s to run the test, it's not exactly long.

I can probably leave them as is, but cut the task list to just 1, instead of 3 tasks.

The problem is that each one needs to be long enough to not be affected by jitter in the strategy, but each one needs to be 3*itself<next to avoid overlaps that make it not deterministic.

So 0.5*3==1.5 but due to jitter we need to ensure there is wiggle room between itself and the next host.

Reply:
Ok. Maybe just moving the first host to zero would be enough. I don't think the first host needs a sleep.

Reply:
Thank you for fixing this.